test-proxy/known_failures.txt

```diff
@@ -1 +1 @@
-
+TestFeatureGap/(traffic_director_enabled|direct_access_requested)
```

Comment:
Actually this format is incorrect.. should be:

TestFeatureGap/traffic_director_enabled\|
TestFeatureGap/direct_access_requested